docs/plugin-protocol/tfplugin5.10.proto

```diff
@@ -412,6 +412,7 @@ message GetProviderSchema {
         map<string, Schema> data_source_schemas = 3;
         map<string, Function> functions = 7;
         map<string, Schema> ephemeral_resource_schemas = 8;
+        map<string, Schema> state_stores = 9;
```

Comment:
```suggestion
        map<string, Schema> state_store_schemas = 9;
```

Reply:
I think the only reason functions are different is because the map value is `Function` and also it represents interface rather than the traditional "schema".

Reply:
TL;DR I'm in favour of `state_store_schemas` here